src/Controller/StudentController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+class StudentController extends AbstractController
+{
+    /**
+     * @Route("/student", name="student")
+     * @param Request $request
+     * @return Response
+     */
+    public function index(Request $request)
+    {
+        $name = $request->query->get('name');
```

Comment:
```suggestion
        $name = $request->get('name');
```
Galima ir trumpiau užrašyti